pdns/dnsdistdist/Makefile.am

```diff
@@ -152,14 +152,13 @@ if HAVE_RE2
 dnsdist_LDADD += $(RE2_LIBS)
 endif
 
-if HAVE_DNS_OVER_TLS
```

Comment:
This would cause `dnsdist` to be linked against `libgnutls` and `$(LIBSSL_LIBS)` when they are available, regardless of whether DNS over TLS support is enabled. If I'm not mistaken you only need to link against `$(LIBCRYPTO_LIBS)` for this to work?

Reply:
yes, that is correct, but I did not find a way to make that happen correctly.